packages/thirdweb/src/react/core/hooks/useTransactionDetails.ts

```diff
@@ -78,12 +82,33 @@ export function useTransactionDetails({
           encode(transaction).catch(() => "0x"),
         ]);
 
-      const [tokenInfo, gasCostWei] = await Promise.all([
+      const account = wallet?.getAccount();
+      if (!account) {
```

Comment:
We need to be able to display the transaction data without a wallet connected.

We should handle it gracefully